launch/tier4_planning_launch/config/scenario_planning/lane_driving/behavior_planning/behavior_velocity_planner/intersection.param.yaml

```diff
@@ -16,6 +16,7 @@
       min_predicted_path_confidence: 0.05
       collision_start_margin_time: 5.0 # [s]
       collision_end_margin_time: 2.0 # [s]
+      frontcar_expected_decel: 1.0 # [m/ss] the expected deceleration of frontcar when frontcar as well as ego are turning
```

Comment:
@soblin 
how about chenge to `assumed_front_car_decel`?
And change explain like 
>Assumed deceleration of front car used to determine whether to stop in the stuck area at the exit of an intersection